tgui/.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
.vscode folder has no effect inside tgui folder, so it should be deleted, and any new settings should be added to the root .vscode folder.

Reply:
Agree, but yarn kept insisting this exists when I run dlx for some reason

Reply:
It doesn't know where vscode root is, and this one will be unused, so it's safe to delete.